### ISSUE TITLE
How do I customize number of epochs in nnunet

### ISSUE THREAD

**Issue by musabaku** [NUBagciLab/PaNSegNet#4]
Default number of epochs is 1000, how do i implement a custom trainer class? Do i need to download nnUnet on my machine and change somethings under /training folder

**Comment by freshman97** [comment]
Yes, if you want to change the epoch, you need to set the max_epoch in the trainer under nnUNet
